server.go

```diff
@@ -1944,6 +1961,40 @@ func SendHeader(ctx context.Context, md metadata.MD) error {
 	return nil
 }
 
+// SetSendCompressor sets a compressor for outbound messages.
```

Comment:
Please document the default behavior if this is not called for a stream, or reference elsewhere if it's documented already.

Reply:
Added the doc
```
// By default, server messages will be sent using the same compressor with which
// request messages were sent.
```